package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "insomnia-plugin-embark-theme",
+  "version": "1.0.2",
+  "description": "A dark theme for the app Insomnia",
```

Comment:
**Nitpick (non-blocking)-** Can we put the project tagline here instead?

"An Insomnia theme for the ambitious"

Reply:
Updated!